### DISCUSSION TITLE
[POLL] What direction would you like to see Catalyst go?

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#409] [Polls]
Pick one.

**Comment by derpygamer2142** [comment]
I also think that you should tone down the advertising for it.

**Reply by jdev082** [reply]
ok

**Reply by PlatesOfSandwiches** [reply]
ok

**Reply by ajskateboarder** [reply]
ok

**Reply by derpygamer2142** [reply]
ok

**Reply by theycallhermax** [reply]
ok

**Comment by jdev082** [comment]
Pleas recommend names for a new project. And what kind of software to create.

**Reply by derpygamer2142** [reply]
Catalyst index: a site with a bunch of different catalysts and their properties
Catalyst IDE: code compiler and editor
CitHub: github but better(or worse)
Cat(alyst) memes: AI generated slideshow of cats

**Reply by derpygamer2142** [reply]
oh wait i voted new name
idfk call it BurritoCorp

**Comment by theycallhermax** [comment]
why is "nowhere" not a option
